xclim/__init__.py

```diff
@@ -1,17 +1,43 @@
 """Climate indices computation package based on Xarray."""
+import logging
+import sys
+import warnings
 from importlib.resources import contents, path
 
-from xclim.core import units  # noqa
+from loguru import logger
+
+from xclim._scripting import enable_synced_logger
+from xclim.core import units
 from xclim.core.indicator import build_indicator_module_from_yaml
 from xclim.core.locales import load_locale
-from xclim.core.options import set_options  # noqa
-from xclim.indicators import atmos, land, seaIce  # noqa
+from xclim.core.options import set_options
+from xclim.indicators import atmos, land, seaIce
 
 __author__ = """Travis Logan"""
 __email__ = "logan.travis@ouranos.ca"
 __version__ = "0.34.2-beta"
 
 
+# On import, clisops sets a root logger for console output. This is huge annoyance to deal with.
+# For upstream fix discussion, see: https://github.com/roocs/clisops/pull/216
+if "clisops" in sys.modules:
+    root_logger = logging.getLogger()
+    root_logger.removeHandler(root_logger.handlers[0])
```

Comment:
xclim doesn't import clisops, I don't think this is the job of xclim?

IMHO, best option is to fix it in clisops, second best is to do this in the user's script, third best is not to use clisops.

Reply:
You're right. I was trying to disable logging from clisops. This type of change is out of scope for this PR. Acceptance of the changes to their logging configuration upstream are still under consideration. Will remove.

Reply:
Removed.